scripts/bulk-access-control.md

```diff
@@ -0,0 +1,93 @@
+# Bulk Access Control Management script
```

Comment:
What is the name of this script?  It cannot be "Bulk Access Control Management" because that has spaces.  I think we need to change the title here & the `Index.md` to have the actual script name.  Maybe it's supposed to be `bulk-access-control` as implied by the name of this file?